src/main/java/org/hydev/mcpm/client/arguments/ArgsParser.java

```diff
@@ -30,20 +30,16 @@
     public ArgsParser(List<CommandParser> allParsers, Consumer<String> log) {
         this.log = log;
 
-        parser = ArgumentParsers
-            .newFor("loader")
-            .build()
-            .defaultHelp(true);
+        parser = ArgumentParsers.newFor("mcpm").build().defaultHelp(true);
```

Comment:
I like the newline formatting. I was gonna comment on how you squished everything to one line on the lower builders but I see you've changed existing stuff too 😦 Not a big deal and definitely nothing close to a merge blocker.

Reply:
oops sorry